tensorflow_lite_support/examples/task/text/desktop/python/cc/BUILD

```diff
@@ -0,0 +1,22 @@
+package(
+    default_visibility = [
+        "//tensorflow_lite_support:users",
+    ],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+# Create the shared library for invoke_nl_classifier
+cc_binary(
+    name = "invoke_nl_classifier",
+    srcs = ["invoke_nl_classifier.cc"],
+    deps = [
+        "//tensorflow_lite_support/cc/port:statusor",
+        "//tensorflow_lite_support/cc/task/core:category",
+        "//tensorflow_lite_support/cc/task/text/nlclassifier:nl_classifier",
```

Comment:
Sure Lu! I will use the C++ demo instead of [invoke_nl_classifier.cc](https://github.com/VihangaAW/tflite-support/blob/c3b71545d12784706d3f088d4c3fe8f0cc298071/tensorflow_lite_support/examples/task/text/desktop/python/cc/invoke_nl_classifier.cc).